api/src/test/java/org/openmrs/module/cohort/api/dao/CohortTypeGenericDaoTest.java

```diff
@@ -79,4 +81,21 @@ public void shouldVoidCohortType() {
 		assertThat(voidedCohortType, nullValue());
 	}
 	
+	@Test
+	public void shouldPurgeCohortType() {
+		CohortType cohortType = dao.get(COHORT_TYPE_UUID);
+		assertNotNull(cohortType);
+		dao.delete(cohortType);
+		CohortType afterAttemptPurge = dao.get(COHORT_TYPE_UUID);
+		assertNull(afterAttemptPurge);
+	}
+	
+	@Test(expected = Exception.class)
```

Comment:
Can we use a more specific exception instead of this generic one?

Reply:
@dkayiwa I updated the test by using ConstraintViolationException.